staff-registration-process.php

```diff
@@ -0,0 +1,38 @@
+<?php 
+require 'connection.php';
+
+$fName = $_POST["firstName"];
+$lName = $_POST["lastName"];
+$email = $_POST["email"];
+$password = $_POST["password"];
+$contact = $_POST["contact"];
+$role = $_POST["role"];
```

Comment:
Don't initialize values like this. First check if those value are assign to the $_POST[] associative array. Values ​​can be assigned to variables only if the condition is true.

Reply:
OK